src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -272,6 +275,19 @@ class MoreInfoLight extends LitElement {
           --md-sys-color-on-surface-rgb: var(--rgb-secondary-text-color);
           --md-sys-color-outline: var(--secondary-text-color);
         }
+        md-outlined-button {
+          --ha-icon-display: block;
+          --md-sys-color-primary: var(--primary-text-color);
+          --md-sys-color-outline: var(--divider-color);
+        }
+        .color-mode {
+          background-image: url("/static/images/color_wheel.png");
+          background-size: cover;
+          --md-sys-color-outline: transparent;
```

Comment:
No outline? It looks a bit weird, but might look worse with the outline...? 🤷‍♂️

Reply:
Mockup
<img width="358" alt="CleanShot 2023-05-11 at 17 40 08@2x" src="https://github.com/home-assistant/frontend/assets/5878303/edc9c561-9f7b-4e9c-bbbb-e776f83b88ae">